packages/ui/src/modal/bid/BidModal.tsx

```diff
@@ -236,6 +236,16 @@ export function BidModal({
 
         const providerOptionsContext = useContext(ProviderOptionsContext)
 
+        const ctaButtonText = providerOptionsContext.disableJumperLink
+          ? copy.ctaBid.length > 0
```

Comment:
hmm not really what I meant. I was just referring to this part:

```
copy.ctaBid
            : token && token.token
            ? 'Make an Offer'
            : trait
            ? 'Make an Attribute Offer'
            : 'Make a Collection Offer'
```
Because the other CTA button isn't shown for the conversion. This is more what I was thinking. Then on line ~779 you can do this:
```
{ providerOptionsContext.disableJumperLink ? ctaButtonText : copy.ctaConvertManually}
```